web/modules/custom/default_content_config/content/node/063eb5b2-3a79-4559-a893-a149cfa2c757.yml

```diff
@@ -31,6 +31,11 @@ default:
   moderation_state:
     -
       value: published
+  path:
```

Comment:
It looks like a bunch of URL alias changes made it into your commits. I can see that in develop, this node already has its alias exported https://github.com/GSA/digital-gov-drupal/blob/develop/web/modules/custom/default_content_config/content/node/063eb5b2-3a79-4559-a893-a149cfa2c757.yml. These files are also conflicting. These are a lot of files and going to be a pain to revert. It might be easiest if your database still has the content changes you did: ie setting the external field:
* git reset --hard HEAD~1
* git fetch
* git merge origin/develop

Then...if you do `./robo.sh drupal-project:export-content` hopefully you just have a few changes and just the external field updates. If not, revert any path alias changes to your files, then commit again. You can then force push your branch back up.

Reply:
@mattsqd I followed these instructions but ran into some issues. [Created a new PR](https://github.com/GSA/digital-gov-drupal/pull/49) to keep things simpler